src/providers/historical-telemetry-provider.js

```diff
@@ -54,6 +54,14 @@ export default class YamcsHistoricalTelemetryProvider {
     request(domainObject, options) {
         options = { ...options };
         this.standardizeOptions(options, domainObject);
+        const isRealtime = this.openmct.time.clock();
+        if ((options.strategy === 'latest') && isRealtime) {
+            console.debug(`🍇 Latest requested in realtime, using cached websocket data`);
```

Comment:
I'll remove these debug statements post review. They're just to help the reviewer.